core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -64,4 +65,63 @@ public void numMaxConnections() {
         final ServerBuilder sb = new ServerBuilder();
         assertThat(sb.maxNumConnections()).isEqualTo(Integer.MAX_VALUE);
     }
+
+    @Test
+    public void setAccessLoggerTest1() {
```

Comment:
- Could you add a test case for the case of using `new VirtualHostBuilder(...).build()`?
- Could you add a test case for the case of `mapper` returning `null`?
- Could you add a Javadoc for the test methods? e.g.
  ```
  /**
   * Makes sure each virtual host can have its custom logger name.
   */
  ```

Reply:
Got it ! thanks